src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -376,18 +376,15 @@
                                      max-decimals  (if crypto-currency? crypto-decimals 2)
                                      regex-pattern (str "^\\d*\\.?\\d{0," max-decimals "}$")
                                      regex         (re-pattern regex-pattern)]
-                                 (when (and (not loading-routes?)
-                                            (re-matches regex new-text))
+                                 (when (re-matches regex new-text)
                                    (debounce/clear-all)
                                    (set-just-toggled-mode? false)
                                    (set-input-state #(controlled-input/add-character % c)))))
        :on-delete            (fn []
-                               (when-not loading-routes?
-                                 (debounce/clear-all)
-                                 (set-just-toggled-mode? false)
-                                 (set-input-state controlled-input/delete-last)))
+                               (debounce/clear-all)
```

Comment:
We should be sure `:wallet/clean-suggested-routes` is called if the value is invalid, I think this line should cover the case but just to be sure https://github.com/status-im/status-mobile/blob/24194fb34e6d9a7a5692a54d7de8882910d3dc9e/src/status_im/contexts/wallet/send/routes/view.cljs#L27